roles/nova-data/tasks/v7k.yml

```diff
@@ -0,0 +1,42 @@
+---
+- block:
+  - name: install multipath-tools packages (ubuntu)
+    package: name={{ item }}
+    with_items:
+      -  multipath-tools
+    register: result
+    until: result|succeeded
+    retries: 5
+
+  - name: generate multipath.conf
+    template: src=etc/multipath.conf dest=/etc/multipath.conf
+              mode=0640
+              owner=root group=root
+
+  - name: restart multipath-tools
```

Comment:
move this to handler and use variable to abstract our service name difference by os.
Follow pattern used by libvirtd service @ https://github.com/blueboxgroup/ursula/blob/master/roles/openstack-meta/defaults/main.yml#L257